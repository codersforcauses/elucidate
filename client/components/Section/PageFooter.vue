<template>
  <footer class="h-20">
    <div class="flex items-center justify-around mb-2 text-sm font-bold">
      <div class="flex justify-center w-full">
        <NuxtLink to="/" class="mx-4">Home</NuxtLink>
        <NuxtLink to="/quiz" class="mx-4">Quiz</NuxtLink>
        <NuxtLink to="/about" class="mx-4">About</NuxtLink>
      </div>
      <div class="flex justify-center w-full">
        <nuxt-img
          src="branding/logo_full.png"
          class="h-12"
          alt="Elucidate Logo"
        />
      </div>
      <div class="flex justify-center w-full">
        <NuxtLink to="/about" class="mx-4">Resources</NuxtLink>
        <NuxtLink to="/" class="mx-4">Contact</NuxtLink>
        <NuxtLink to="/quiz" class="mx-4">Sponsors</NuxtLink>
      </div>
    </div>
    <hr />
    <div class="flex items-center justify-center py-8 text-4xl text-gray-400">
      <a href="https://www.facebook.com/AtarSurvivalGuide/">
        <font-awesome-icon :icon="['fab', 'fa-facebook']" class="mx-4" />
      </a>
      <a href="https://www.instagram.com/atar_survival_guide/">
        <font-awesome-icon :icon="['fab', 'fa-instagram']" class="mx-4" />
      </a>
      <a href="https://www.youtube.com/channel/UC3d8MUFzDD01IoeJCDazSpg">
        <font-awesome-icon :icon="['fab', 'fa-youtube']" class="mx-4" />
      </a>
      <a href="https://www.linkedin.com/company/atar-survival-guide/">
        <font-awesome-icon :icon="['fab', 'fa-linkedin']" class="mx-4" />
      </a>
    </div>
  </footer>
</template>

<script>
export default {
  name: 'PageFooter',
};
</script>
