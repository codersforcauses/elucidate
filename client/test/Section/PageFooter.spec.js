import { mount } from '@vue/test-utils';
import PageFooter from '@/components/Section/PageFooter.vue';
describe('PageFooter', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(PageFooter);
    expect(wrapper.vm).toBeTruthy();
  });

  test('renders correctly', () => {
    const wrapper = mount(PageFooter);
    expect(wrapper.element).toMatchSnapshot();
  });
});
