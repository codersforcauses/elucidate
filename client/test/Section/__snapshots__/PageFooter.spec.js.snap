// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter renders correctly 1`] = `
<footer
  class="h-20"
>
  <div
    class="flex items-center justify-around mb-2 text-sm font-bold"
  >
    <div
      class="flex justify-center w-full"
    >
      <nuxtlink
        class="mx-4"
        to="/"
      >
        Home
      </nuxtlink>
       
      <nuxtlink
        class="mx-4"
        to="/quiz"
      >
        Quiz
      </nuxtlink>
       
      <nuxtlink
        class="mx-4"
        to="/about"
      >
        About
      </nuxtlink>
    </div>
     
    <div
      class="flex justify-center w-full"
    >
      <nuxt-img
        alt="Elucidate Logo"
        class="h-12"
        src="branding/logo_full.png"
      />
    </div>
     
    <div
      class="flex justify-center w-full"
    >
      <nuxtlink
        class="mx-4"
        to="/about"
      >
        Resources
      </nuxtlink>
       
      <nuxtlink
        class="mx-4"
        to="/"
      >
        Contact
      </nuxtlink>
       
      <nuxtlink
        class="mx-4"
        to="/quiz"
      >
        Sponsors
      </nuxtlink>
    </div>
  </div>
   
  <hr />
   
  <div
    class="flex items-center justify-center py-8 text-4xl text-gray-400"
  >
    <a
      href="https://www.facebook.com/AtarSurvivalGuide/"
    >
      <font-awesome-icon
        class="mx-4"
        icon="fab,fa-facebook"
      />
    </a>
     
    <a
      href="https://www.instagram.com/atar_survival_guide/"
    >
      <font-awesome-icon
        class="mx-4"
        icon="fab,fa-instagram"
      />
    </a>
     
    <a
      href="https://www.youtube.com/channel/UC3d8MUFzDD01IoeJCDazSpg"
    >
      <font-awesome-icon
        class="mx-4"
        icon="fab,fa-youtube"
      />
    </a>
     
    <a
      href="https://www.linkedin.com/company/atar-survival-guide/"
    >
      <font-awesome-icon
        class="mx-4"
        icon="fab,fa-linkedin"
      />
    </a>
  </div>
</footer>
`;
